pelican/plugins/_utils.py

```diff
@@ -122,4 +122,4 @@ def get_plugin_name(plugin):
     if inspect.ismodule(plugin):
         return plugin.__name__
 
-    return type(plugin).__qualname__
+    return type(plugin).__qualname__
```

Comment:
Last commit seems to have removed an EOF marker for some reason. Could you fix this?